enigma-js/test/integrationTests/template.03_deploy_fail_wrong_eth_address.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable require-jsdoc */
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import forge from 'node-forge';
+import Web3 from 'web3';
+import Enigma from '../../src/Enigma';
+import utils from '../../src/enigma-utils';
+import EnigmaContract from '../../../build/contracts/Enigma';
+import EnigmaTokenContract from '../../../build/contracts/EnigmaToken';
+import VotingETHContract from '../../../build/contracts/VotingETH';
+import * as eeConstants from '../../src/emitterConstants';
+import data from '../data';
+import EthCrypto from 'eth-crypto';
+
+
+forge.options.usePureJavaScript = true;
+
+function sleep(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+describe('Enigma tests', () => {
+  let accounts;
+  let web3;
+  let enigma;
+  let votingETHContract;
+  let epochSize;
+  it('initializes', () => {
+    const provider = new Web3.providers.HttpProvider('http://localhost:9545');
+    web3 = new Web3(provider);
+    return web3.eth.getAccounts().then((result) => {
+      accounts = result;
+      enigma = new Enigma(
+        web3,
+        EnigmaContract.networks['4447'].address,
+        EnigmaTokenContract.networks['4447'].address,
+        'http://localhost:3346',
+        {
+          gas: 4712388,
+          gasPrice: 100000000000,
+          from: accounts[0],
+        },
+      );
+      enigma.admin();
+      expect(Enigma.version()).toEqual('0.0.1');
+    });
+  });
+
+  it('initializes VotingETH contract', async () => {
```

Comment:
You don't need this section here, because you are making up your own wrong address. The contract is already initialized in the migrations.

Reply:
Addressed in 5752f96